README.md

```diff
@@ -1,3 +1,22 @@
+Domain Model for Acceptance Criteria
+| Objects | Properties           | Messages                       | Outputs  |
+|---------|----------------------|--------------------------------|----------|
+| Airport | apronPlanes @Array   | landPlane(@plane, @weather)    | @void    |
+|         | apCap @Integer       | returnApCap()                  | @Integer |
+|         |                      | setApCap(@Integer)             | @Integer |
+|         |                      | isAirportFull()                | @Boolean |
+|         |                      | takeoffPlane(@Plane, @Weather) | @void    |
+|         |                      | isPlaneOnApron(@planeId)       | @Boolean |
+| Plane   | id @String           | returnId()                     | @String  |
+
+Domain Model for Extended Acceptance Criteria
+| Objects   | Properties             | Messages                         | Outputs    |
+| --------- | ---------------------- | -------------------------------- | ---------- |
+| Weather   |                        | getWeather()                     | @String    |
+
+* Not sure why it hasn't created a new table for the extended criteria following the same layout as the previous table
+** Likely due to difference in width of "Messages" section and difference in number of rows
```

Comment:
Comment: See Code

Reply:
At this point I had attempted to add spaces between the "[...]------[...]" as the markdown "cheat sheet" had; however, this didn't appear to fix the problem I was having and I figured it was due probably to the width of columns and number of rows.
I realise now that I could have incorporated both the Acceptance Criteria and Extended Acceptance Criteria into one domain model.